test_files/1_22_333_file.txt

```diff
@@ -0,0 +1 @@
+122333
```

Comment:
Nits: I think `simple_case_1.txt`, `simple_case_2.txt`, etc would be sufficient instead of 1_22_333_file.txt, a_b_file.txt, aaabbb_file.txt.